Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "linux-loader"
+version = "0.1.0"
+authors = ["cathy <cathy.zhang@intel.com>"]
```

Comment:
I think this needs to be your complete name.

Reply:
Fixed